scripts/generate-types-for-not-supported-ts-versions.mjs

```diff
@@ -15,10 +15,12 @@ limitations under the License.
 */
 
 import * as fs from 'node:fs';
+
+import packageJSON from '../package.json' assert { type: 'json' };
+
 import { getTypeFilesInformation } from './shared/types-info.mjs';
 // generate warning when running with Node 18
 // (node:75278) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
```

Comment:
**nitpick:** This comment must be moved with "import packageJSON..." as it relates to it